cli/src/commit_templater.rs

```diff
@@ -300,9 +300,15 @@ fn build_commit_keyword_opt<'repo>(
             maybe_entries.map_or(true, |entries| !entries.contains(commit.id()))
         })),
         "conflict" => language.wrap_boolean(wrap_fn(property, |commit| {
+            if let MergedTreeId::Merge(tree_ids) = commit.tree_id() {
+                return !tree_ids.is_resolved();
+            }
```

Comment:
nit: maybe we can add `commit.has_conflict()`? `conflict()` revset can also be optimized.

```
cli/src/commit_templater.rs
306:            commit.tree().unwrap().has_conflict()
lib/src/default_revset_engine.rs
872:            commit.tree().unwrap().has_conflict()
cli/src/commands/git.rs
908:        if commit.tree()?.has_conflict() {
```

Reply:
Good idea! Done. The commit is still pretty small and simple, but let me know if you prefer that I split it out.